### PR TITLE
Better Salary Slip logs in Process Payroll

### LINKED ISSUE #7223: Make a nicer Process Payroll Log
- [x] Use Bootstrap Tables
- [x] Show Employee Name and Total Pay

Current Log

screen shot 2016-12-14 at 6 14 24 pm

### PR DESCRIPTION
**Pull request by KanchanChauhan**
Fixed https://github.com/frappe/erpnext/issues/7223

screen shot 2016-12-26 at 4 18 13 pm

### CODE REVIEW

**Comment by rmehta** [comment]
Screenshot?

**Comment by KanchanChauhan** [comment]
@rmehta Added

File: erpnext/hr/doctype/process_payroll/process_payroll.py [hunk 1]
```diff
@@ -103,17 +103,46 @@ def create_salary_slips(self):
...
+		else:
+			log = frappe.render_template(self.get_log_template(),
+						dict(ss_list=ss_list,
+							keys=sorted(ss_list[0].keys()),
+							title=_('Created Salary Slips')))
+			return log
+
+	def	get_log_template(self):
+		return '''
+		<table class="table">
```
Comment (rmehta) [inline erpnext/hr/doctype/process_payroll/process_payroll.py]: you can add this to a template file in /templates/inludes

**Comment by rmehta** [comment]
👍 

- [ ] Currency formatting
- [ ] Is the Employee "name" correct (?) I see email addresses

**Comment by KanchanChauhan** [comment]
@rmehta 
screen shot 2016-12-26 at 5 13 29 pm

**Review by rmehta — APPROVED** [review]

**Pull request merged**